pages/aptos/get-price.mdx

```diff
@@ -0,0 +1,65 @@
+import Input from "../../components/Input";
+import Arg from "../../components/Arg";
+import { InputFormats } from "../../utils/InputFormat";
+import Example from "../../components/Example";
+import Examples from "../../components/Examples";
+import DynamicCode from "../../components/DynamicCode";
+import EvmCall from "../../components/EvmCall";
+import { Tab, Tabs } from "nextra-theme-docs";
+
+# Get Price
+
+Get the latest price and confidence interval for the requested price feed id.
+The price feed id is a 32-byte id written as a hexadecimal string; see the [price feed ids](https://pyth.network/developers/price-feed-ids) page to look up the id for a given symbol.
```

Comment:
Maybe export the next 4 lines as a component and reuse them everywhere?

Reply:
this is a little annoying to do right now because of how the markdown formatting works (if you make a component, you have to write html), but totally agree we should have a way to share reused text. Will think about it and set something up.